cheddar/src/lib.rs

```diff
@@ -187,11 +189,12 @@ impl Contract {
     }
 
     #[payable]
-    /// terminate vesting before the cliff
-    /// burn the tokens
-    pub fn terminate_vesting(&mut self, account_id: &AccountId) {
+    /// Cancels token allocation in a vesting account. All not vested tokens
+    /// will be burned.
+    /// Only owner can call this function.
+    pub fn cancel_vesting(&mut self, account_id: &AccountId) {
         assert_one_yocto();
-        self.assert_minter(env::predecessor_account_id());
+        self.assert_owner_calling();
```

Comment:
we had a security issue here. In the previous version any minter could cancel any vesting account (including the genesis vesting!)